cosmos/templates/index.html

```diff
@@ -137,4 +137,34 @@ <h5 class="card-title">{{ news.title }}</h5>
         </div>
     </div>
 </section>
+<section id="partners" class="pb-5">
+    <div class="container py-5">
+        <div class="section-title">
+            <h2>Partners</h2>
+        </div>
```

Comment:
Is it not cleaner without adding a whole section, and just having a row of logos?